.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: php
-
+dist: trusty
```

Comment:
I should mention that actually, all the build on HHVM may fail due to its unavailability on Ubuntu Precise. So, all Travis configuration files need to be updated.